tests/baselines/reference/objectRest.symbols

```diff
@@ -1,42 +1,42 @@
 === tests/cases/conformance/types/rest/objectRest.ts ===
-let o = { a: 1, b: 'no' }
->o : Symbol(o, Decl(objectRest.ts, 0, 3))
+var o = { a: 1, b: 'no' }

```

Comment:
I changed the source declaration (line 2) so I could reuse `o` in a destructuring assignment below.